src/liquid/components/ld-tabs/ld-tab.tsx

```diff
@@ -0,0 +1,79 @@
+import '../../components' // type definitions for type checks and intelliSense
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Prop,
+  Watch,
+} from '@stencil/core'
+
+/**
+ * @virtualProp ref - reference to component
+ * @virtualProp {string | number} key - for tracking the node's identity when working with lists
+ */
+@Component({
+  tag: 'ld-tab',
+  styleUrl: 'ld-tab.css',
+  shadow: false,
+})
+export class LdTab {
+  @Element() el: HTMLElement
+
+  /**
+   * If present, this boolean attribute indicates that the tab is selected.
+   */
+  @Prop({ mutable: true, reflect: true }) selected?: boolean
+
+  /**
+   * Disables the tab.
+   */
+  @Prop() disabled?: boolean
+
+  /**
+   * Emitted with the id of the selected tab.
+   */
+  @Event() tabChange: EventEmitter<string>
+
+  private handleTabClick(ev) {
+    ev.preventDefault()
+
+    if (this.disabled) return
+
+    if (ev.currentTarget.getAttribute('aria-selected')) return
+
+    this.selected = true
+  }
+
+  @Watch('selected')
+  emitEvent(newSelected: boolean, oldSelected: boolean) {
+    if (!newSelected || newSelected === oldSelected) return
+
+    const index = Array.prototype.indexOf.call(
+      this.el.closest('.ld-tablist__scroll-container').children,
+      this.el
+    )
+
+    this.tabChange.emit(index)
+  }
+
+  render() {
+    return (
+      <button
+        class="ld-tab"
+        role="tab"
+        aria-selected={this.selected ? 'true' : undefined}
+        aria-disabled={this.disabled ? 'true' : undefined}
```

Comment:
Why didn't you let the user decide whether he wants to use `disabled` or `aria-disabled` (just like we did it with the `<ld-toggle />` element)?

Reply:
Good question. Actually I am not a big fan of the disabled property at all. No one should use it. Let's take the button as an example: If you apply the disabled prop on the button, the user can not focus it anymore via the tab key. A visually impaired user would then potentially have problems understanding what is going on, because he doesn't "see" the button. Putting aria-disabled on the button instead and preventing the action via JavaScript on the other hand fixes this problem. Now we still want to support the disabled prop on the button, because we are still offering a button CSS component and we just want to make sure that if a developer uses this prop, the button is styled accordingly. With the -ld-tabs component though we do not have a CSS component (yet) so we can decide by ourselves whether a tab should support both props or just one. For the sake of simplicity (and for the edge case where the current tab is selected but also disabled) I went with aria-disabled.

Reply:
This is very opinionated. I understand your arguments and they make sense to me. But a strongly opinionated library may not be adapted as well as a less opinionated library. But this question is probably a matter of taste, anyway. Feel free to resolve this conversation.